src/codegeneration/generator/CPSTransformer.js

```diff
@@ -164,7 +165,13 @@ export class CPSTransformer extends TempVarTransformer {
     super(identifierGenerator);
     this.reporter = reporter;
     this.stateAllocator_ = new StateAllocator();
-    this.labelSet_ = Object.create(null);
+
+    // This is currently a Map<string, LabelState> where the key is the
+    // label.name. We should probably change this to Set<LabelState> but that
+    // requires us depending on a real Set which might be too slow in old JS
+    // engines.
```

Comment:
I just don't want to depend on a Set polyfill. I could create a LabelSet class which would be fast today but it would be another ~100 loc. WDYT?